components/liquidity/ExitPoolForm.tsx

```diff
@@ -236,10 +234,10 @@ const ExitPoolForm = ({
                     } else if (value <= 0) {
                       return "Value cannot be zero or less";
                     } else if (
-                      poolStatus.toLowerCase() === "active" &&
+                      market.status.toLowerCase() !== "resolved" &&
```

Comment:
Are you sure this is correct? I've discussed this with Malte previously, I can double check though

Reply:
Checked with him, what you've changed is correct